lablib/lib/imageio.py

```diff
@@ -283,33 +283,17 @@ def format_string(self) -> str:
     @property
     def hash_string(self) -> str:
         """:obj:`str`: A sequence representation used for ``oiiotool`` arguments formatting."""  # noqa
-        frame: ImageInfo = min(self.frames)
+        frame: ImageInfo = min(self.imageinfos)
         ext: str = frame.extension
         basename = frame.name.split(".")[0]
 
         result = f"{basename}.{self.start_frame}-{self.end_frame}#{ext}"
         return result
 
-    @property
-    def format_string(self) -> str:
-        """:obj:`str`: A sequence representation used for ``ffmpeg`` arguments formatting.  # noqa
-
-        Error:
-            That's a duplicate so let's run tests and remove it.
-        """
-        frame: ImageInfo = min(self.frames)
-        ext: str = frame.extension
-        basename = frame.name.split(".")[0]
-
-        result = f"{basename}.%0{self.padding}d{ext}"
-        return result
-
     @property
     def padding(self) -> int:
         """:obj:`int`: The sequence's frame padding."""
-        frame = min(self.frames)
-        result = len(str(frame.frame_number))
-        return result
+        return len(str(self.start_frame))
```

Comment:
Shouldn't we rather use end frame here? It usually has the higher padding value (ex. 999-1020)

Reply:
so so true!
will fix when i get home today 👌